src/main/kotlin/org/rust/lang/core/resolve/NameResolution.kt

```diff
@@ -293,6 +296,26 @@ fun processMacroSimpleResolveVariants(element: RsMacroBodySimpleMatching, proces
     return simple.any { processor(it) }
 }
 
+fun processMetaItemResolveVariants(element: RsMetaItem, processor: RsResolveProcessor): Boolean {
+    if (element.parentOfType<RsMetaItem>()?.identifier?.text != "derive") return false
+    val traitName = element.identifier.text
+    val stdTrait = STD_DERIVE_TRAITS[traitName]
```

Comment:
Looks like it can through an exception if `traitName` is not in `STD_DERIVE_TRAITS`.